servers/lib/tsconfig.json

```diff
@@ -25,7 +26,7 @@
     "types": ["node", "jest"], //use node and jest types
     "typeRoots": [
       "node_modules/@types" //use node_modules/@types for type definitions
-    ],
+    ]
```

Comment:
new version of the prettier seems to be adding this by default. Can you please try `yarn format` and see if this comma is being kept/removed?

Reply:
It seems to be removed